BookingsApi/BookingsApi.DAL/Helper/CloneHearingToCommandMapper.cs

```diff
@@ -61,10 +62,32 @@ public static class CloneHearingToCommandMapper
                 };
             }).ToList();
 
+            var hearingParticipants = hearing.Participants.Where(x => x.LinkedParticipants.Any()).ToList();
+            var linkedParticipantDtos = new List<LinkedParticipantDto>();
+            foreach (var hearingParticipant in hearingParticipants)
```

Comment:
There's a fair bit going on here. It would be easier to follow to wrap this part into a private method.